File: src/tests/Markify.Fixtures/Projects/Source/Class/ClassProperties.cs
public abstract class AbstractFooType
{
    public virtual int SealedProperty { get; set; }
}

public class FooType : AbstractFooType
{
    public int AutoProperty { get; set; }

    public int ReadOnlyProperty { get; }

    public int WithExpressionBody => 1;

    public int WriteOnlyProperty
    {
        set { }
    }

    public int WithGetterModifierProperty
    {
        internal get { return 1; }
        set { }
    }

    public int WithSetterModifierProperty
    {
        get { return 1; }
        internal set { }
    }

    public int WithInitialValueProperty { get; } = 1;

    protected int WithModifierProperty { get; set; }

    int WithNoModifierProperty { get; set; }

    protected internal int WithMultipleModifiersProperty { get; set; }

    public static int StaticProperty { get; set; }

    public virtual int VirtualProperty { get; set; }
    
    public sealed override int SealedProperty { get; set; }
}

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Class/ClassProperties.vb
Public MustInherit Class AbstractFooType
    Public Overridable Property SealedProperty() As Integer
End Class

Public Class FooType
    Inherits AbstractFooType

    Public Property AutoProperty()

    Public ReadOnly Property ReadOnlyProperty() As Integer

    Public WriteOnly Property WriteOnlyProperty() As Integer
        Set
        End Set
    End Property

    Public Property WithGetterModifierProperty() As Integer
        Friend Get
            Return 1
        End Get
        Set(value As Integer)
        End Set
    End Property

    Public Property WithSetterModifierProperty() As Integer
        Get
            Return 1
        End Get
        Friend Set(value As Integer)
        End Set
    End Property

    Public ReadOnly Property WithInitialValueProperty() As Integer = 1

    Protected Property WithModifierProperty() As Integer

    Property WithNoModifierProperty() As Integer

    Protected Friend Property WithMultipleModifiersProperty() As Integer

    Public Shared Property StaticProperty() As Integer

    Public Overridable Property VirtualProperty() As Integer

    Public NotOverridable Overrides Property SealedProperty() As Integer
End Class